tests/BiosTests.hs

```diff
@@ -4,9 +4,11 @@
 module Main where
 
 import Test.Tasty
-import Test.Tasty.ExpectedFailure
 import Test.Tasty.HUnit
 import Test.Hspec.Expectations
+#if __GLASGOW_HASKELL__ < 810
+import Test.Tasty.ExpectedFailure
+#endif
```

Comment:
Why does this change make sense?

Reply:
Fixes a redundant import warning on ghc 8.10